examples/SCM/src/helper_funcs.jl

```diff
@@ -0,0 +1,524 @@
+using NCDatasets
```

Comment:
Many of the tools implemented in this PR are still being tested, and therefore should not be in the main branch of a published package. In addition, EKP.jl should be a generic optimizer and have lightweight examples. This PR is well out of that scope, since the example requires a lot of extra source code. Now that EKP.jl has been published as a package, let's move all this to another package within CliMA where we can have all the tools we need to use EKP to train the EDMF from LES/observations.